modules/basicfuncs/numeric-funcs.c

```diff
@@ -21,11 +21,66 @@
  *
  */
 
+#include <math.h>
+
 typedef gboolean (*AggregateFunc)(gpointer, gint64);
 
+#define VALUE_TYPE_GINT64  1
+#define VALUE_TYPE_GDOUBLE 2
+
+struct value
+{
+  int value_type;
+  union
+  {
+    gint64 gint64;
+    gdouble gdouble;
+  } value_data;
+};
```

Comment:
`value` is an awful name.  Any suggestion for a better name is welcome :wink:, the idea here is to be able to distinguish if we parsed and integer or a float, and store the value in an easy to manipulate format.